build.gradle

```diff
@@ -153,7 +158,24 @@ task renameShadowJar(type: Copy, group: 'release', dependsOn: [shadowJar]) {
 task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
+task prepareGameEngineProperties(group: 'release') {
+    inputs.file gameEnginePropertiesFile
+    outputs.file gameEnginePropertiesArtifactFile
+
+    doLast {
+        copy {
+            from gameEnginePropertiesFile
+            into gameEnginePropertiesArtifactFile.parent
+        }
+        ant.propertyfile(file: gameEnginePropertiesArtifactFile) {
+            entry key: 'engine_version', value: version
+        }
+    }
+}
+
+task prepareRootFiles(group: 'release', dependsOn: [prepareGameEngineProperties]) {}
```

Comment:
Does it make sense to have a task require a single other task without any action itself?